packages/js/rollup.config.js

```diff
@@ -45,7 +45,8 @@ export default [
     external: ['http', 'https', 'url', 'os', 'fs', 'util', 'domain', 'async_hooks'],
     output: {
       file: pkg.main,
-      format: 'cjs'
+      format: 'cjs',
+      sourcemap: true
```

Comment:
Why this?

Reply:
No. I put it here to help me with debugging. I don't remember if it did, but there's no harm letting it there.